packages/azure-services/src/ioc-types.ts

```diff
@@ -24,6 +24,8 @@ export type QueueServiceURLProvider = () => Promise<ServiceURL>;
 
 export const cosmosContainerClientTypes = {
     A11yIssuesCosmosContainerClient: 'a11yIssuesCosmosContainerClient',
+    ScanBatchesCosmosContainerClient: 'scanBatchesCosmosContainerClient',
+    ScanRunsCosmosContainerClient: 'scanRunsCosmosContainerClient',
```

Comment:
>   ScanRunsCosmosContainerClient: 'scanRunsCosmosContainerClient', [](start = 1, length = 66)

remove this or OnDemandPageScanRunResults

Reply:
updated